### PR TITLE
feat(chinese): simple chinese draft example

### PR DESCRIPTION
**Pull request by alexcg1**
We've had many folks ask for a Chinese example, so I've tried converting the wikipedia sentence search. However, upon indexing it chokes on the first line of `headlines.txt` (created by `get-data.sh`)

Would be great if you could have a look at it. I'm likely missing something obvious.

### Notes

- Using Chinese news dataset from Kaggle, headlines only
- Using `bert-base-chinese` in `pods/encode.yml`

### CODE REVIEW

File: drafts/chinese-text/pods/encode.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+!TransformerTorchEncoder
+with:
+  pooling_strategy: auto
+  pretrained_model_name_or_path: bert-base-chinese
+  max_length: 96 default
```
Comment (deepampatel) [inline drafts/chinese-text/pods/encode.yml:5]: @alexcg1  seems this line should be 
```suggestion
  max_length: 96
```

**Comment by Yongxuanzhang** [comment]
The example could work. Thanks for your great work! @alexcg1 

1. I changed the folder and the data. The previous dataset is a little bit serious.
2. Since this one is almost the same as the Wikipedia example, we don't have to add a test for this one.
3. I will update the Readme then. The question is if we want to have a Chinese Readme for this one?

File: chinese-text-search/README_CN.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  </tr>
+  <tr>
+    <td>
+      Jina 版本
+    </td>
+    <td>
+      1.0.0
+    </td>
+  </tr>
+</table>
```
Comment (nan-wang) [inline chinese-text-search/README_CN.md:28]: Give it a try to native markdown table. Currently, it looks wired.
image

File: chinese-text-search/app.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
```
Comment (nan-wang) [inline chinese-text-search/app.py:1]:
```suggestion
__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
```

File: chinese-text-search/README_CN.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+请遵循如下规范对Docker镜像进行命名，否则无法上传Jina hub。
+
+```
+jinahub/type.kind.image-name:image-version-jina_version
+```
+
+比如:
+
+```
+jinahub/app.example.wikipedia-sentences-30k:0.2.8-0.9.23
```
Comment (nan-wang) [inline chinese-text-search/README_CN.md:104]:
```suggestion
jinahub/app.example.chinese-text-search:0.0.1-1.0.0
```

File: chinese-text-search/README_CN.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+## 上传到 [Jina Hub](https://github.com/jina-ai/jina-hub)
+
+1. 确保安装 `pip install jina[hub]`
```
Comment (nan-wang) [inline chinese-text-search/README_CN.md:109]:
```suggestion
1. 确保安装 `pip install jina[hub]==1.0.0`
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
For README file, please follow the same naming convention as https://github.com/jina-ai/jina/tree/master/.github/i18n

File: chinese-text-search/tests/test_wikipediasearch.py [hunk 6]
```diff
@@ -0,0 +1,82 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
```
Comment (bwanglzu) [inline chinese-text-search/tests/test_wikipediasearch.py:1]: same, 2021

**Review by bwanglzu — CHANGES_REQUESTED** [review]
Great work! Please use jina 1.x

File: chinese-text-search/app.py (hunk 3 above)
Comment (Yongxuanzhang) [reply chinese-text-search/app.py]: done

File: chinese-text-search/tests/test_wikipediasearch.py (hunk 6 above)
Comment (Yongxuanzhang) [reply chinese-text-search/tests/test_wikipediasearch.py:1]: I prefer to remove the tests folder. Since it's duplicate to current test

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: chinese-text-search/README_CN.md [hunk 7]
```diff
@@ -101,11 +93,11 @@ jinahub/type.kind.image-name:image-version-jina_version
 比如:
 
 ```
-jinahub/app.example.wikipedia-sentences-30k:0.2.8-0.9.23
+jinahub/app.example.chiese-text-search:0.0.1-1.0.0
```
Comment (alexcg1) [inline chinese-text-search/README_CN.md:83]: spelling

File: chinese-text-search/README_CN.md [hunk 8]
```diff
@@ -1,41 +1,33 @@
...
-</table>
-
+| 列表        | 内容           |
+| ------------- |:-------------:|
+| 输入     | 一个text文本文件 |
+| 输出     | top_k 个匹配查询的句子| 
+| Jina版本 | 1.0.0   |
+
+
+- [基于Transformer的中文文字搜索](#--transformer-------)
```
Comment (alexcg1) [inline chinese-text-search/README_CN.md]: Should this be `Transformers`?

File: chinese-text-search/README_CN.md [hunk 9]
```diff
@@ -1,41 +1,33 @@
...
-    </td>
-  </tr>
-  <tr>
-    <td>
-      Jina 版本
-    </td>
-    <td>
-      1.0.0
-    </td>
-  </tr>
```
Comment (alexcg1) [inline chinese-text-search/README_CN.md]: I think we can remove this? It'll be a pain to maintain

**Review by nan-wang — APPROVED** [review]

**Pull request merged**